taurus_pyqtgraph/taurusmodelchoosertool.py

```diff
@@ -264,7 +264,7 @@ def dropMimeData(self, data):
             models.append(data.text())
 
         if len(models) > 0:
-            conf_list = []
+            conf_list = self.getTaurusPlotDataItemsConf()
```

Comment:
just a reminder for looking at this carefully: we need yet to define the expected behaviour when setting/ adding models in tpg.

Possibilities:
- a) replace all existing models (this is what `...tpg.TaurusPlot.setModel` and the previous behavior of the drop do
- b) add models unconditionally (this is what would be done by a drop after this change). This would duplicate curves if they already exist
- c) add models only if they are not already added (this is what `...qwt5.TaurusPlot.addModel` or a drop in the old qwt5 widgets do)

I, personally think that the best is to implement addModel with strategy "b" in the tpg's modelchooser, and delegate to it in `tpg.TaurusPlot.addModel` as well as for the drops.